crates/aptos/src/node/mod.rs

```diff
@@ -37,11 +37,7 @@ use tokio::time::Instant;
 ///
 #[derive(Parser)]
 pub enum NodeTool {
-    AddStake(AddStake),
-    UnlockStake(UnlockStake),
-    WithdrawStake(WithdrawStake),
-    IncreaseLockup(IncreaseLockup),
-    RegisterValidatorCandidate(RegisterValidatorCandidate),
+    InitializeValidator(InitializeValidator),
```

Comment:
Should we add InitializeOwner here as well? People might or might use the CLI for that but we likely would for local/devnet/internal testnet testing

Reply:
InitializeStakeOwner is in the stake command, not node.

(I mentioned I am calling it InitializeStakeOwner here, as I find InitializeOwnerOnly pretty confusing. I can change to InitializeStakeOwner in the move code too, if that sounds good to you)